package.json

```diff
@@ -28,6 +28,7 @@
     "dev:listings": "cd services/listings && yarn dev",
     "dev:backend": "cd backend/core && yarn dev",
     "dev:all": "concurrently \"yarn dev:listings\" \"yarn dev:app:public\"",
+    "dev:all:nest": "concurrently \"yarn dev:backend\" \"yarn dev:app:public\"",
```

Comment:
I think we should probably update `dev:all` rather than adding a new command - my understanding is that we're phasing out the "old" JSON based backend anyway?

Reply:
Addressed through a more significant refactor.